vagrant/provision.sh

```diff
@@ -31,9 +31,16 @@ curl -Ss https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
 # fetch the metadata from the new repo
 apt-get update
 
-# install postgres 11 and go. build-essential is required for cgo
-apt-get install -y --no-install-recommends build-essential postgresql-11 golang-go
-export PATH=$PATH:/root/go/bin
+# install postgres 14 and go. build-essential is required for cgo
+apt-get install -y --no-install-recommends build-essential postgresql-14
+cd /tmp
+wget https://dl.google.com/go/go1.17.linux-amd64.tar.gz
+sudo tar -xvf go1.17.linux-amd64.tar.gz
+sudo mv go /usr/local
+
+export GOROOT=/usr/local/go
+export GOPATH=$HOME/go
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
 
 # install sql-migrate
 go get -v github.com/rubenv/sql-migrate/...
```

Comment:
Unchanged, but this line no longer works for me, which breaks vagrant provisioning 😢 


```
root@ubuntu-bionic:/# go get -v github.com/rubenv/sql-migrate/...
github.com/go-gorp/gorp (download)
package github.com/go-gorp/gorp/v3: cannot find package "github.com/go-gorp/gorp/v3" in any of:
        /usr/lib/go-1.10/src/github.com/go-gorp/gorp/v3 (from $GOROOT)
        /root/go/src/github.com/go-gorp/gorp/v3 (from $GOPATH)
package crypto/ed25519: unrecognized import path "crypto/ed25519" (import path does not begin with hostname)
```

Reply:
Actually please ignore this - I was on a bad commit, this was caused by an old go version in the machine